processes/procstat_test.go

```diff
@@ -138,7 +138,7 @@ func TestGetStats(t *testing.T) {
 
 			for procName, instances := range results {
 
-				So(procName, ShouldResemble, "mockProcName")
+				So(procName, ShouldResemble, "systemd-hostnamed^@")
```

Comment:
Process name was gathered previously from `stat` mock (/proc/\<pid\>/stat) and was "mockProcName", now is is gathered from `cmdline` which was already set to "systemd-hostnamed^@", so i updated this check